lib/bootloader_setup.pm

```diff
@@ -472,20 +473,25 @@ sub bootmenu_network_source {
             }
 
             select_installation_source({m_protocol => $m_protocol, m_mirror => $m_mirror});
-
-            my $remote = get_var("REMOTE_TARGET");
-            if ($remote) {
-                my $dns = get_host_resolv_conf()->{nameserver};
-                push @params, get_var("NETSETUP") if get_var("NETSETUP");
-                push @params, "nameserver=" . join(",", @$dns);
-                push @params, ("$remote=1", "${remote}password=$password");
-            }
         }
     }
     type_string_very_slow(" @params ");
     return @params;
 }
 
+sub bootmenu_remote_target {
```

Comment:
We have call of bootmenu_network_source in `lib/bootloader_spvm.pm`, I guess it's not used there, but I would not change it in this PR.

Reply:
so, I will leave as it is and will remember that.